packages/api-server/src/base/env-config.ts

```diff
@@ -6,7 +6,6 @@ dotenv.config({ path: "./.env" });
 export const envConfig = {
   databaseUrl: getRequired("DATABASE_URL"),
   godwokenJsonRpc: getRequired("GODWOKEN_JSON_RPC"),
-  newRelicLicenseKey: getOptional("NEW_RELIC_LICENSE_KEY"),
```

Comment:
let's keep this variable in envConfig? since `.env` in `README.md` has such config?

Reply:
Updated in another PR:
https://github.com/nervosnetwork/godwoken-web3/pull/444/files#diff-e5de419b6edadcf4d59c1cf7d75c9d50d2a567039753911c63d7eec4aaf0c75bR9

```diff
-  newRelicLicenseKey: getOptional("NEW_RELIC_LICENSE_KEY"),
+  _newRelicLicenseKey: getOptional("NEW_RELIC_LICENSE_KEY"),
```